### PR TITLE
Heroine buds no longer chop the hairstyle of anyone wearing them

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
fix: Heroine bugs no longer make people appear partially bald when wearing them.
/:cl:

Also I renamed the moth clothing snack to a more descriptive name if anyone
gets one by accident.

### CODE REVIEW

**Comment by adamkad1** [comment]
moth clothing snack?

**Comment by coiax** [comment]
Moths can eat clothing. Because they're moths.

**Comment by adamkad1** [comment]
moth balls when?

**Comment by GhostEcho** [comment]
Any chance of this fix getting applied to the Chamelon Hats? The berets in particular have this issue from my experience

**Comment by ExcessiveUseOfCobblestone** [comment]
we should just make "" the default and generic can be for when we can't be arsed to make it nice for items that need it.

As a long-haired static it pains me to no end I look like a friar when wearing a bunch of hats.

File: code/modules/research/xenobiology/crossbreeding/_clothing.dm [hunk 1]
```diff
@@ -100,6 +100,14 @@ Slimecrossing Armor
...
 	icon_state = "peaceflower"
 	item_state = "peaceflower"
+	slot_flags = ITEM_SLOT_HEAD
+	body_parts_covered = NONE
+	dynamic_hair_suffix = ""
+	force = 0
+	throwforce = 0
+	w_class = WEIGHT_CLASS_TINY
+	throw_speed = 1
+	throw_range = 3
```
Comment (ShizCalev) [inline code/modules/research/xenobiology/crossbreeding/_clothing.dm]: Hmmmmmmmmmm

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**